src/libs/Environment/Environment.js

```diff
@@ -12,7 +19,20 @@ function isDevelopment() {
     return lodashGet(Config, 'ENVIRONMENT', CONST.ENVIRONMENT.DEV) === CONST.ENVIRONMENT.DEV;
 }
 
+/**
+ * Get the URL based on the environment we are in
+ *
+ * @returns {Promise<unknown>}
```

Comment:
Is the type really unknown? (I'm not sure if I have seen this before)

Reply:
Oh that was something phpstorm autoinjected and i missed it. Updated the code to clean this up.